perenv.spec.js

```diff
@@ -90,3 +90,70 @@ describe("Object signature", () => {
     );
   });
 });
+
+describe("loadPerEnvMap", () => {
+  it("Should use the assignment identifier as the import namespace", () => {
+    process.env.FEATURE_VERSION = "alpha";
+
+    const result = applyMacro(`
+      import { loadPerEnvMap } from "./perEnv.macro";
+      const feature = loadPerEnvMap({alpha: './aaaa.js'}, 'FEATURE_VERSION');
+    `);
+
+    expect(result.includes("import * as feature ")).toBe(true);
+  });
+
+  it("Should throw if the envar value is not found in the env map", () => {
```

Comment:
Why?  Should not assign as null ?
Why should fail ?

Reply:
Without the nullable flag set to true, it should throw an error.

This is for the cases where you want to guarantee that something in the map has been applied.

As I am typing this, I think that perhaps the default behavior is backwards though

instead of `loadPerEnvMap(map, envar, ?nullable)` it should maybe be `loadPerEnvMap(map, envar, ?required)` as a way to make sure that something can be imported.